### PR TITLE
CANS 1179 rebuild base test image

### PR DESCRIPTION
**Pull request by TylerOSI**
## Description
This fixes some base image dependencies and upgrades chrome to the version 74.

upgraded:
* chrome
* chrome-driver
* selenium-webdriver
* capybara

notes:
* tests start a new browser session for each test case
* caregiver name could not be set to empty string for some reason

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters for the project.
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by TylerOSI** [comment]
Jenkins test this please

**Comment by TylerOSI** [comment]
related to #668

**Comment by TylerOSI** [comment]
related to #674

File: spec/regression/case_worker_scenarios_spec.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
   current_date = Time.now.getlocal.strftime('%m/%d/%Y')
   assessment_link = 'CANS Assessment Form'
 
-  before(:all) do
+  before(:each) do
```
Comment (denys-davydov) [inline spec/regression/case_worker_scenarios_spec.rb]: Can't find what has changed that we need to login before each scenario.

**Comment by TylerOSI** [comment]
Jenkins test this please

File: spec/regression/case_worker_scenarios_spec.rb (hunk 1 above)
Comment (TylerOSI) [reply spec/regression/case_worker_scenarios_spec.rb]: I can do some research to try to find out. It started after i updated chrome, chrome driver, and related dependencies.

Comment (TylerOSI) [reply spec/regression/case_worker_scenarios_spec.rb]: seems like this is due to how [Capybara is hooked into Rspec](https://github.com/teamcapybara/capybara/blob/4da9195870d7782bce3d3b0db27e0f793ea24343/lib/capybara/rspec.rb#L18). it clears the session after each test run, which brings the user back to login. We can monkey patch this code by removing `require capybara/rspec` and configuring Rspec to connect to Capybara ourselves. Though there are performance benefits for doing that, I don't think its worth it.

**Comment by denys-davydov** [comment]
Retest this please

**Comment by denys-davydov** [comment]
The user story is postponed so closing the PR

**Pull request closed without merging**